spec/models/news_item_spec.rb

```diff
@@ -1,5 +1,19 @@
 require 'rails_helper'
 
 RSpec.describe NewsItem, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe 'attribute validations' do
+    it "does not validate when 'published_by' is not defined" do
+      news_item = NewsItem.new(published_by: nil)
+      expect(news_item).to be_invalid
+      expect(news_item.errors.messages.keys).to include :published_by
+    end
+  end
+
+  context 'when a news_item is published' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.